docs/migration/migrate_7_0.asciidoc

```diff
@@ -9,6 +9,13 @@ your application to Kibana 7.0.
 
 See also <<release-highlights>> and <<release-notes>>.
 
+[float]
+=== Removed support for user's relying on direct index privileges to the Kibana index in Elasticsearch
+*Details:* With the introduction of Kibana RBAC in 6.4, user's no longer require privileges to the Kibana index in Elasticsearch. Instead, users 
+should be granted <<kibana-privileges>>. Prior to 7.0, when a user that relies upon direct index privileges logs into Kibana, a deprecation warning is logged. If you are using the `kibana_user` role to grant access to Kibana, or a custom role using <<kibana-privileges>>, no changes are required.
+
+*Impact:* You must change any roles which grant access to Kibana using index privileges to instead use <<kibana-privileges>>.
```

Comment:
It's definitely implied, but what do you think about calling out reporting jobs here? Jobs with auth setup will need to ensure that those accounts are updated as well. Otherwise, they'll start failing with a non-descriptive "forbidden" error.

Reply:
> Jobs with auth setup will need to ensure that those accounts are updated as well. Otherwise, they'll start failing with a non-descriptive "forbidden" error.

Are you referring to pending jobs that are using the legacy roles at the time of upgrade? Or is there some other scenario which I'm not thinking of?

Reply:
I don't care so much about the pending jobs, as they should be negligible. I was more concerned about configured watcher actions. Users can configure Watcher alerts to [include a Kibana report as an email attachment](https://www.elastic.co/guide/en/elastic-stack-overview/6.5/actions-email.html).